src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -500,9 +513,9 @@ bool isSupportedConversion(Conversion conversion, BoundExpression operandOpt)
                 {
                     case ConversionKind.Identity:
                     case ConversionKind.DefaultOrNullLiteral:
-                        return true;
                     case ConversionKind.ImplicitReference:
-                        return operandOpt?.IsLiteralNullOrDefault() == true;
+                    case ConversionKind.ImplicitNullable:
```

Comment:
>case ConversionKind.ImplicitNullable: [](start = 20, length = 37)

This feels suspicious. ```ImplicitNullable``` doesn't imply type identity. Why would the slot be reusable?

Reply:
Removed.

---
In reply to: [250782645](https://github.com/dotnet/roslyn/pull/32729#discussion_r250782645) [](ancestors = 250782645)